pages/verifier/verify.vue

```diff
@@ -35,13 +35,17 @@
     <div class="w-72 bg-gray-100 p-6 bg-gray-300 -mt-8">
       <h2 class="text-xl font-semibold">Updates</h2>
       <ol class="list-decimal">
-        <li v-for="message in logMessages">{{ message }}</li>
+        <li v-for="message in logMessages">
+          [{{ message.source }}<p v-if="message.target"> -> {{ message.target }}</p>] {{ message.message }}
```

Comment:
```suggestion
          [{{ message.source }}<p v-if="message.target === undefined"> -> {{ message.target }}</p>] {{ message.message }}
```

Reply:
I guess here you meant `message.target !== undefined `